paymentintent.go

```diff
@@ -114,14 +114,16 @@ type PaymentIntentListParams struct {
 
 // PaymentIntentLastPaymentError represents the last error happening on a payment intent.
 type PaymentIntentLastPaymentError struct {
```

Comment:
Technically, this should be `Error` right? Is it worth changing this and releasing as a major version to be cleaner?